test/acceptance/accessibility.py

```diff
@@ -243,6 +243,78 @@ def test_multiple_rubrics(self):
         self._check_a11y(self.staff_area_page)
 
 
+class MultipleOpenAssessmentA11yTest(OpenAssessmentA11yTest, FullWorkflowMixin):
+    """
+    Test accessibility when we have a unit containing multiple ORA blocks.
+    """
+
+    def setUp(self):
+        super(MultipleOpenAssessmentA11yTest, self).setUp('multiple_ora', staff=True)
+        self.staff_area_page = StaffAreaPage(self.browser, self.problem_loc)
+
+    # TODO: remove this method override. See TNL-1593
+    def _check_a11y(self, page):
+        page.a11y_audit.config.set_scope(
+            exclude=[
+                ".container-footer",
+                ".nav-skip",
+                "#global-navigation",
+                "#openassessment",
+                "#openassessment__message"
+                "#openassessment__steps"
+                "#openassessment__response"
+                "#openassessment__student-training"
+                "#openassessment__peer-assessment"
+                "#openassessment__staff-assessment"
+                "#openassessment__grade"
+            ],
+        )
+        page.a11y_audit.config.set_rules({
+            "ignore": [
+                "color-contrast",  # TODO: AC-198
+                "empty-heading",  # TODO: AC-197
+                "link-href",  # TODO: AC-199
+                "link-name",  # TODO: AC-196
+                "skip-link",  # TODO: AC-179
+            ]
+        })
+        page.a11y_audit.check_for_accessibility_errors()
+
+    def check_a11y(self):
+        """ check accessibility"""
+        self._check_a11y(self.submission_page)
+        self._check_a11y(self.self_asmnt_page)
+        self._check_a11y(self.peer_asmnt_page)
+        self._check_a11y(self.student_training_page)
+        self._check_a11y(self.staff_asmnt_page)
+        self._check_a11y(self.grade_page)
+        self._check_a11y(self.staff_area_page)
+
+    def set_vert_no(self, vert_no):
+        """ Set vertical number """
+        self.submission_page.vert_no = vert_no
+        self.self_asmnt_page.vert_no = vert_no
+        self.peer_asmnt_page.vert_no = vert_no
+        self.student_training_page.vert_no = vert_no
+        self.staff_asmnt_page.vert_no = vert_no
+        self.grade_page.vert_no = vert_no
+        self.staff_area_page.vert_no = vert_no
+
+    def test_multiple_ora_complete_flow(self):
+        """
+        Test accessibility when we have a unit containing multiple ORA blocks.
+        """
+        self.set_vert_no(0)
+        # First Vertical - which is selected by default
+        learner = self.do_train_self_peer()
+        self.check_a11y()
+
+        # Second Vertical
+        self.set_vert_no(1)
+        self.do_train_self_peer()
```

Comment:
The comment I had for acceptance tests applies here too - ensure that the problem is fully completed before checking a11y.

We probably ought to check it before starting as well, to ensure that the learner's "submit a response" view is a11y compliant.